### DISCUSSION TITLE
Is it possible to change the email address?

### DISCUSSION THREAD

**Discussion by RyansSohn** [seriousm4x/UpSnap#427] [Help wanted]
Hi, Is it possible to change the email address of the admin account?

**Comment by seriousm4x** [comment]
Yes, it's possible, but only with the backend UI.
Open `localhost:8090/_/` login with your admin credentials -> bottom left profile icon -> manage admins -> and change the email there.

**Reply by RyansSohn** [reply]
Thanks
